server/package.json

```diff
@@ -14,6 +14,7 @@
     "@types/cors": "^2.8.9",
     "@types/express": "^4.17.11",
     "@types/mongoose": "^5.10.3",
+    "@types/morgan": "^1.9.2",
```

Comment:
nice, thanks for this! FYI @Teddarific , @jjanelee97 added the missing types for morgan in your server application